src/database/schema/ChannelListMemberSchema.ts

```diff
@@ -68,10 +68,15 @@ class ChannelListMemberSchema implements BaseDbSchema {
   };
 
   saveIfNotExist = async (db: SQLiteDatabase): Promise<void> => {
-    const ifExists = await this.checkIfExist(db);
-    if (ifExists) return;
+    try {
+      const ifExists = await this.checkIfExist(db);
+      if (ifExists) return;
 
-    this.save(db);
+      this.save(db);
+    } catch (e) {
+      console.log('saveIfNotExist error', e);
+      throw e;
+    }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The error handling added here is a good start, but it could be improved by providing more context in the error message. Instead of just logging 'saveIfNotExist error', consider including additional information about the current state that might help diagnose the issue. Also, `this.save(db)` is an asynchronous operation and should be awaited to ensure proper execution order.

```diff
    try {
      const ifExists = await this.checkIfExist(db);
      if (ifExists) return;
 
-75:       this.save(db);
+75:       await this.save(db);
    } catch (e) {
-77:       console.log('saveIfNotExist error', e);
+77:       console.log(`Error in saveIfNotExist. Does record exist? ${ifExists}`, e);
      throw e;
    }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->